app/locale/vi.coffee

```diff
@@ -33,7 +33,7 @@ module.exports = nativeDescription: "Tiếng Việt", englishDescription: "Vietn
 #    classroom_in_box_blurb1: "Engage every student so that they believe coding is for them."
 #    classroom_in_box_blurb2: "Empower any educator to feel confident when teaching coding."
 #    classroom_in_box_blurb3: "Inspire all school leaders to create a world-class computer science program."
-#    classroom_in_box_blurb4: ""
+#    classroom_in_box_blurb4: "" # only for koudashijie.com
```

Comment:
remove `# only for koudashijie.com`

Reply:
This is the exact copy from [en.coffee](https://github.com/codecombat/codecombat/blob/master/app/locale/en.coffee#L35) I realized vi.coffee is a bit out of date compare with the English version, hence the difference and some changes